src/vm/vm.c

```diff
@@ -81,4 +81,14 @@ void syscall(VirtualMachine *vm) {
     syscall_print(vm);
     break;
   }
+}
+
+#include "asm/ast.h"
```

Comment:
maybe move the reg_to_string(...) function to vm.c so the asm parsing stuff and actual virtual machine stay seperate